### PR TITLE
Update hrv_OCRFixReplaceList.xml

### PR DESCRIPTION
**Pull request by diomed**

### CODE REVIEW

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 1]
```diff
@@ -1376,8 +1394,8 @@
     <RegEx find="pelova" replaceWith="rica" />
     <RegEx find="([sS])pase(\b|[mšt])" replaceWith="$1pasi$2" />
     <RegEx find="([sS])pasen([aeiou])" replaceWith="$1pašen$2" />
-    <RegEx find="spolja?š?nj?([aeiu])" replaceWith="vanjsk$1" />
-    <RegEx find="Spolja?š?nj?([aeiu])" replaceWith="Vanjsk$1" />
+    <RegEx find="spolja?š?nj?" replaceWith="vanjsk" />
```
Comment (diomed) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: @xylographe how do I know if questionmark in the end is for letter or for whole word?

Comment (ivandrofly) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: For word: `(\w+)?`
For letter/digit \w? or simple `[a-zA-Z]?`

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 2]
```diff
@@ -40,6 +40,9 @@
     <Word from="ceo" to="cijeli" />
     <Word from="Ceo" to="Cijeli" />
     <Word from="cepa" to="cijepa" />
+    <Word from="cvetu" to="cvijetu" />
+    <Word from="cvjetu" to="cvijetu" />
```
Comment (ivandrofly) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: `<Word from="cvj?etu" to="cvijetu" />` maybe?

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 3]
```diff
@@ -436,6 +441,10 @@
     <Word from="suština" to="bit" />
     <Word from="suštinski" to="bitni" />
     <Word from="svestan" to="svjestan" />
+    <Word from="svjest" to="svijest" />
+    <Word from="Svjest" to="Svijest" />
+    <Word from="svjesti" to="svijesti" />
+    <Word from="Svjesti" to="Svijesti" />
```
Comment (ivandrofly) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: `<Word from="([Ss])vjesti?" to="$1vijesti" />`

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 4]
```diff
@@ -76,6 +79,10 @@
     <Word from="deo" to="dio" />
     <Word from="Deo" to="Dio" />
     <Word from="dešava" to="događa" />
+    <Word from="dete" to="dijete" />
+    <Word from="djete" to="dijete" />
+    <Word from="Dete" to="Dijete" />
+    <Word from="Djete" to="Dijete" />
```
Comment (ivandrofly) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: Line 82 and 83 can be merged.

<Word from="([dD])j?ete" to="$1ijete" />

Ps: Ignore https://github.com/SubtitleEdit/subtitleedit/pull/1959#pullrequestreview-487002

File: Dictionaries/hrv_OCRFixReplaceList.xml (hunk 1 above)
Comment (xylographe) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: The question mark applies to the preceding **atom**,

- **`ab(cd|de)?`** : the sub-expression `(cd|de)` is the atom with highest priority (due to the parenthesis). The expression matches `abcd`, `abde`, and `ab`.
- **`ab[cde]?`** : the class `[cde]` is the atom. The expression matches `abc`, `abd`, `abe`, and `ab`.
- **`ab?cd?`** : the letters `b` and `d` are atoms. The expression matches `abcd`, `abc`, `acd`, and `ac`.

Comment (diomed) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: so, like this?:
`<RegEx find="spolja?š?n[j]?" replaceWith="vanjsk" /> `

File: Dictionaries/hrv_OCRFixReplaceList.xml (hunk 2 above)
Comment (xylographe) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: @ivandrofly These are `<WholeWords>`, not `<RegularExpressions>`.

File: Dictionaries/hrv_OCRFixReplaceList.xml (hunk 4 above)
Comment (diomed) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: I know you meant well, @ivandrofly , but if you look closely, these words are all in a **wordlist**, not in regex, so it's not really possible to merge them like that.

File: Dictionaries/hrv_OCRFixReplaceList.xml (hunk 1 above)
Comment (xylographe) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: A class with only one letter is the same as just the letter, `[j]` equals `j`.

```xml
<RegEx find="spolja?š?nj?" replaceWith="vanjsk" />
```

File: Dictionaries/hrv_OCRFixReplaceList.xml (hunk 3 above)
Comment (ivandrofly) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: @xylographe Why not replacing these 3-5 lines with a single regex?

**Pull request closed without merging**